docs/settings.md

```diff
@@ -28,12 +28,21 @@ For example, in case you want to run the app on port `5000`, just set the enviro
 
 ## Development
 
-* `--reload` - Enable auto-reload.
+* `--reload` - Enable auto-reload. Uvicorn supports two versions of auto-reloading behavior enabled by this option. See below for important differences between them.
```

Comment:
I can't remember if it was this project that had auto creation of CLI flags using docs, so I'd be weary with the "See below" and replace the last sentence with an explicit explanation. But Kludex would know.

Reply:
I think I can eliminate the relative reference anyway. Let me try….

Reply:
Resolved by subsequent commit.